src/main/java/com/k3ntako/HTTPServer/controllers/SimplePost.java

```diff
@@ -8,17 +8,32 @@
 
 public class SimplePost implements ControllerInterface {
   private FileIOInterface fileIO;
+  private UUIDInterface uuid;
 
-  public SimplePost(FileIOInterface fileIO) {
+  public SimplePost(FileIOInterface fileIO, UUIDInterface uuid) {
     this.fileIO = fileIO;
+    this.uuid = uuid;
   }
 
-  public Response getResponse(RequestInterface request) throws IOException {
+  public Response getResponse(RequestInterface request) throws IOException, HTTPError {
     var body = request.getBody();
-    Path path = FileSystems.getDefault().getPath("./data/simple_post.txt");
+    validateBody(body);
+
+    var fileUUID = uuid.generate();
+    var filePath = "./data/" + fileUUID + ".txt";
+    Path path = FileSystems.getDefault().getPath(filePath);
 
     fileIO.write(path, body);
 
-    return new Response();
+    var response = new Response();
+    response.setBody(fileUUID);
+
+    return response;
+  }
+
+  private void validateBody(String body) throws HTTPError {
```

Comment:
This is awesome! If we are going full MVC this validation should happen in the model.